packages/toolboxes/cosmos/src/toolbox/kujira.ts

```diff
@@ -59,7 +59,7 @@ export const KujiraToolbox = ({ server }: ToolboxParams = {}): KujiraToolboxType
           .filter(({ denom }) => {
             if (!denom || denom.includes("IBC/")) return false;
 
-            return denom === USK_KUJIRA_FACTORY_DENOM || !denom.startsWith("FACTORY");
+            return denom === USK_KUJIRA_FACTORY_DENOM || YUM_KUJIRA_FACTORY_DENOM || !denom.startsWith("FACTORY");
```

Comment:
Code doesn't work that way - it will return true always because of that

Reply:
How would you do it? This part was quite vague to me

Reply:
@GiMa-Maya that needs to be coded as previous comparison `denom ===` 😅 that's how JS/TS works